xml/hwbp_registers.xml

```diff
@@ -59,6 +63,11 @@
             7: The trigger is a trigger source external to the TM.  The
             remaining bits in this register act as described in \RcsrTmexttrigger.
 
+            8: This trigger was disabled. In this state, \RcsrTdataTwo and
```

Comment:
I think that we should avoid allocating a new type if we don't have to.  Can 15 serve this purpose?  15 could mean the trigger exists and is either disabled or not currently available.

Reply:
Done.